conf/documentation.conf

```diff
@@ -1397,6 +1397,13 @@ description=<<EOT
 The virtual router id for keepalive. Leave untouched unless you have another keepalive cluster in this network.
 EOT
 
+[active_active.dns_on_vip_only]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Activate so the clients in registration and isolation are pointed to the VIP of the server instead of all the servers.
```

Comment:
I would make this more explicit. E.g. set the name server  option in dhcp replies to point only to the VIP in cluster mode rather than to all servers in the cluster.